src/main/java/step2/Operation.java

```diff
@@ -0,0 +1,72 @@
+package step2;
+
+public class Operation {
+
+    private static final String[] OPERATORS = {"+", "-", "*", "/"};
+
+    public Operation() {
+    }
+
+    public int calculate(String expression) throws IllegalArgumentException {
+
+        String[] ops = expression.split(" ");
+
+        Integer operand = convert(ops[0]);
+        String operator = null;
+        for (int i = 1; i < ops.length; i++) {
+            if (operator == null) {
+                operator = ops[i];
+                continue;
+            }
+
+            Integer value = convert(ops[i]);
+            if (operand != null && operator != null) {
+                operand = getValue(operator, operand, value);
+                operator = null;
+            }
+        }
+
+        return operand;
+    }
+
+    private int convert(String s) {
+        if (s == null || s.isEmpty()) {
+            throw new IllegalArgumentException("Input number is null or empty");
+        }
+        return Integer.parseInt(s);
+    }
+
+    private int getValue(String operator, int a, int b) {
+        switch (operator) {
+            case "+":
+                return add(a, b);
+            case "-":
+                return subtract(a, b);
+            case "*":
+                return multiply(a, b);
+            case "/":
+                return divide(a, b);
+            default:
+                throw new IllegalArgumentException("Possible operators are [" + OPERATORS + "]");
```

Comment:
이렇게 작성하는 경우 메세지는 아래와 같이 출력될 것 같네요.

```
Possible operators are [java.lang.String]
```

정확한 메세지는 아닌데, 예를 들면 위와 같이 객체의 참조가 나올 것 같아요.

원하시는 로직을 구현하려면 아래와 같이 변경이 필요해보입니다~

```suggestion
throw new IllegalArgumentException("Possible operators are [" + String.join(",", OPERATORS) + "]");
```

Reply:
앗 놓친 부분이네요.. 확인 감사합니다.
enum으로 유효 오퍼레이터 검증 로직 옮기면서, 아래 코드로 대체하였습니다. 
`Optional<StringOperator> so = Arrays.stream(StringOperator.values())
            .filter(stringOperator -> stringOperator.operator.equals(operator))
            .findFirst();
        return so.orElseThrow(() -> new IllegalArgumentException(
            "Possible operators are [" + Arrays.stream(StringOperator.values()).
                map(o -> o.operator).
                collect(Collectors.toList()) + "]"));`